### DISCUSSION TITLE
Problem with setting effect properties that are Expression Controllers

### DISCUSSION THREAD

**Discussion by wmwccdd** [inlife/nexrender#977] [Q&A]
My template has a few things controlled by some expression controllers like sliders and checkboxes. So for a simple example, lets say I have a layer called CTRL and there is a slider control named BoxWidth that drives the width of a shape layer rectangle. I'm trying to pass a value to that slider using this:

"assets": [
      {
        "type": "data",
        "layerName": "CTRL",
        "property": "Effect.BoxWidth.Slider",
        "value": 1
      },
    ]

I would get this error:
Can't find a property sequence BoxWidth.Slider for key: BoxWidthwithin layer: CTRL

The idea being that I have several other things connected to the box width, and I'm trying to keep driving them with this one slider.  Any thoughts would be appreciated!

**Comment by wmwccdd** [comment]
Actually got this working today by renaming the layer to Controller instead of CTRL. Not sure why that did the trick but it did.

**Reply by tolgaburcak** [reply]
Try Effects instead of Effect
"property": "Effects.BoxWidth.Slider",

**Reply by wmwccdd** [reply]
Yeah sorry my post here contained a typo. The actual JSON said Effects. The only thing that fixed this issue, for some weird reason was to rename the layer Controller instead of CTRL.

**Comment by volition74** [comment]
I had basically the same thing happen to me. Trying to update a psuedo effect slider i had done previously plenty of times now.

For a particular project/comp/layer nexrender just did not like it! Spitting out basically the same format error message.
```
'Error: nexrender: Can't find a property sequence <Trim_Paths_Control.Offset Speed> for key: <Trim_Paths_Control>within layer: <Control Null>'
```
Luckily I found this post as I was at a loss I had tried a few renaming things as I wondered if it was a parsing thing. But i was having no luck. First I tried renaming my "Control Null" to "Controller" which was similar to the OPs naming problems but that didn't help me.

So I probably overdid the renaming. I renamed the comp, created a brand new layer I then created 2 Sliders with silly names lol and I also renamed the project. I was tired of trying things so I exaggerated it and it worked.I tried renaming from "Control Null" to "Controller" But that didn't work.

I genuinely do not know what fixed it out of the number of things i tried.. It must be  a parsing thing.  And it's too late right now for me to try and go back to 1 change at a time to identify what was the precise fix for me. If i get some spare time later I might as it might be handy to keep a record of the similar problems and what fixed it. to help diagnose an issue. 

I tested this the next day. Just saving as a new project name fixed mine. So my advice is try renaming the layer and/or the project name.
